composer.json

```diff
@@ -139,6 +156,10 @@
                 "package-webentry": "2.18.0",
                 "packages": "^0"
             },
+            "docker-executors": {
+                "docker-executor-cdata": "1.0.1",
+                "docker-executor-node-ssr": "1.5.0"
```

Comment:
@ryancooley @caleeli This probably needs to go to the `require` section, because is not only an enterprise related package.

Reply:
microservice versions are not handled by composer